### DISCUSSION TITLE
How to get a Pro version for free for open source contributors?

### DISCUSSION THREAD

**Discussion by Syammed2429** [community/community#146643] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

Hi Guys,
Is there's anywhere to get a free Pro version for the open source contributors??

**Comment by MindfulLearner** [comment, marked as answer]
Hey man!  

This question has already been answered here:  
[GitHub Community Discussion 53661](https://github.com/orgs/community/discussions/53661)

Check out the first part of this guide:  
[GitHub Developer Program Documentation](https://docs.github.com/en/get-started/exploring-integrations/github-developer-program)

If you're a student, you can apply directly here:  
[GitHub Education Program](https://github.com/education)

Have a nice day!
